File: app/src/main/java/com/tangtown/org/view/TextViewDialog.java
package com.tangtown.org.view;

import android.os.Bundle;
import android.view.View;

import com.app.org.base.BaseActivity;
import com.app.org.view.dialog.BaseDialogLoading;
import com.github.anzewei.parallaxbacklayout.ParallaxBack;
import com.tangtown.org.R;

/**
 * Created by lixingxing on 2018/3/8.
 */
@ParallaxBack(edge = ParallaxBack.Edge.LEFT,layout = ParallaxBack.Layout.PARALLAX)
public class TextViewDialog extends BaseActivity{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.text_dialog);
    }

    public void onBtnClick(View view) {
        switch (view.getId()) {
            case R.id.BaseLoadingDialog:
                BaseDialogLoading baseDialogLoading = new BaseDialogLoading(baseContext);
                baseDialogLoading.show();
                break;
            default:
                break;
        }
    }
}

File: app/src/main/java/com/qqkjbasepro/org/view/TextViewDialog.kt
package com.qqkjbasepro.org.view

import android.content.Intent
import android.os.Bundle
import android.view.View

import com.app.org.base.BaseActivity
import com.app.org.view.dialog.BaseDialogLoading
import com.github.anzewei.parallaxbacklayout.ParallaxBack
import com.qqkjbasepro.org.R
import kotlinx.android.synthetic.main.text_dialog.*

/**
 * Created by lixingxing on 2018/3/8.
 */
@ParallaxBack(edge = ParallaxBack.Edge.LEFT, layout = ParallaxBack.Layout.PARALLAX)
class TextViewDialog : BaseActivity() {
    override fun setRootView() {
    }

    override fun initView() {
    }

    override fun getData() {
    }

    override fun initDefaultData(intent: Intent?) {
    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.text_dialog)
        setupTitleLayout(baseTitle,false)
        baseTitle.title = "测试"
    }

    fun onBtnClick(view: View) {
        when (view.id) {
            R.id.BaseLoadingDialog -> {
                val baseDialogLoading = BaseDialogLoading(baseContext)
                baseDialogLoading.show()
            }
            else -> {
            }
        }
    }
}
